Tools/justin/disassemble.py

```diff
@@ -0,0 +1,35 @@
+import os
+import posixpath
+import re
+
+HERE = os.path.dirname(__file__)
+ROOT = os.path.join(HERE, "../..")
+THIS = os.path.relpath(__file__, ROOT).replace(os.path.sep, posixpath.sep)
+
+STENCILS = os.path.relpath(os.path.join(ROOT, "Python/jit_stencils.h"))
+assert os.path.isfile(STENCILS), f"Stencil file {STENCILS} doesn't exist"
+
+
+stencil_h = open(STENCILS).read()
+bytesmatch = r"// ([A-Z_]+)\nstatic unsigned char [^\n]+\[\] = \{([^\}]+)\};"
+stencilbytes = re.findall(bytesmatch, stencil_h, re.DOTALL)
+stencilbytes = {x:bytes(eval(f"[{y}]")) for x,y in stencilbytes}
+
+from capstone import *
```

Comment:
It would be good to include a script to set up a venv or a  `requirements.txt` .